models.py

```diff
@@ -216,12 +214,14 @@ def update(self, user_update: User) -> bool:
             values = self.cur.fetchall()
             values = values[0]
             dict_from_db = dict(zip(keys, values))
+            dict_user = asdict(user_update)
+            dict_user_update = {k: v for k, v in dict_user.items() if v is not None}
             dict_for_db = {**dict_from_db, **dict_user_update}
```

Comment:
dict_user, dict_user_update, dict_for_db. Why do you use this and not using just data class object?